Sources/Stripe/Helpers/ActionType.swift

```diff
@@ -7,7 +7,7 @@
 //
 
 public enum ActionType: String, Codable {
-    case charge
+    case applicationFee = "application_fee"
     case stripeFee = "stripe_fee"
-    case none
+    case tax
```

Comment:
Is there no raw value here? Maybe for consistency add in `case tax = "tax"`

Reply:
@anthonycastelli There only needs to be a raw value for camelCase properties because foundation doesn't support snake_case encoding strategy yet on linux so they won't synthesize unless I provide the value that's coming from Stripe.

Reply:
Right, right, but for the same of readability, wouldn't specifying the raw value there be a bit cleaner

Reply:
Not necessarily I think. I think more on the lines of this has to be explicit because of limitations where as if all enums had their default case even when they don't need them it could signal that people should do it regardless and they might miss the goal of having Codable values and having things work automatically for them. But that's just my thought.

Reply:
Thats true. I guess I just prefer to have everything expressed is one thing is 😝 

Pass on this one then. We can leave it as thats a good point about it "magically" working